doc/man3/SSL_CTX_set_cert_verify_callback.pod

```diff
@@ -34,10 +34,10 @@ argument I<arg> is specified by the application when setting I<callback>.
 I<callback> should return 1 to indicate verification success and 0 to
 indicate verification failure. If SSL_VERIFY_PEER is set and I<callback>
 returns 0, the handshake will fail. As the verification procedure may
-allow to continue the connection in case of failure (by always returning 1)
-the verification result must be set in any case using the B<error>
-member of I<x509_store_ctx> so that the calling application will be informed
-about the detailed result of the verification procedure!
+allow continuation of the connection in the case of failure (by always
```

Comment:
Perhaps "allow the connection to continue in the case of failure"?

Reply:
Much better, thanks.